build.sbt

```diff
@@ -10,15 +10,14 @@ lazy val publishSettings = Seq(
 
 lazy val standardSettings = Defaults.coreDefaultSettings ++ publishSettings ++ Seq(
   organization := "io.sphere",
-  scalaVersion := "2.12.8",
-  crossScalaVersions := Seq("2.11.12", "2.12.8"),
+  scalaVersion := "2.13.0-M5",
```

Comment:
This could be extracted to a val.

Reply:
👍 
https://github.com/sphereio/sphere-scala-libs/pull/96/commits/3543550ec9d93884fe9ff4e63b101fa2b86eab7e